vignettes/backtesting.Rmd

```diff
@@ -166,7 +167,7 @@ forecasts %>%
     doctor_visits %>%
       epix_as_of(doctor_visits$versions_end) %>%
       group_by(geo_value) %>%
-      tidyr::fill(percent_cli),
+      fill(percent_cli),
```

Comment:
This doesn't seem to do what was intended.  Maybe some sort of `complete` was intended beforehand?

```r
waldo::compare(
doctor_visits %>%
      epix_as_of(doctor_visits$versions_end) %>%
      group_by(geo_value) %>%
      fill(percent_cli)
doctor_visits %>%
      epix_as_of(doctor_visits$versions_end) %>%
      `attr<-`("sorted", NULL) %>%
      `attr<-`(".internal.selfref", NULL) %>%
      group_by(geo_value)
)
#> ✔ No differences
```
It does unset some attrs that shouldn't be there... that's [epiprocess#561](https://github.com/cmu-delphi/epiprocess/issues/561)

Reply:
I'm not sure where this was introduced. Hopefully it gets handled in #431 @dsweber2